src/components/AmplitudeAnalytics/constants.ts

```diff
@@ -38,6 +38,8 @@ export enum WALLET_CONNECTION_RESULT {
   FAILED = 'Failed',
 }
 
+export const NATIVE_CHAIN_ADDRESS = 'NATIVE'
```

Comment:
`NATIVE_CHAIN_ID`? it's not really an address so kinda confusing otherwise. open to other options also

Reply:
im ok w NATIVE_CHAIN_ID, changed.